ntbs-service/Models/Validations/ValidationAttributes.cs

```diff
@@ -9,16 +9,12 @@ namespace ntbs_service.Models.Validations
 {
     public class ValidDateAttribute : RangeAttribute
     {
-        private readonly string StartDate;
         public ValidDateAttribute(string startDateString) : base(typeof(DateTime),
-            startDateString, DateTime.Now.ToShortDateString())
-        {
-            StartDate = startDateString;
-        }
+            startDateString, DateTime.Now.ToShortDateString()) {}

```

Comment:
Has this changed the underlying validation too? If so was this intentional?

Reply:
I see now that it hasn't, but it does seem weird that we don't have a message indicating that it has to be after 2010. Is this definitely what Nancy/Tehreem wanted?

Reply:
Yeah Nancy and Tehreem wanted the message to just mention the upper bound as they thought the lower bound part of the date message could be confusing (especially with the 1900 ones not just the 2010 ones)